src/pages/ChooseArea.svelte

```diff
@@ -38,10 +38,7 @@
   };
 
   let showAbout = false;
-  const params = new URLSearchParams(window.location.search);
-  if (params.get("schema")) {
-    schemaStore.set(params.get("schema") as Schema);
-  }
+  let params = new URLSearchParams(window.location.search);
```

Comment:
any particular reason for this change? Don't we prefer const where we can use it?

Reply:
Oops, just habit from jumping languages constantly. (Rust uses `let` and `let mut`). Will revert. Could be nice to set up eslint or similar in the future to enforce consistency